src/Framework/Microsoft.AspNetCore.App.props

```diff
@@ -19,6 +19,8 @@
     <Dependency Include="Microsoft.AspNetCore.Authentication"                          Version="$(MicrosoftAspNetCoreAuthenticationPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Authorization.Policy"                    Version="$(MicrosoftAspNetCoreAuthorizationPolicyPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Authorization"                           Version="$(MicrosoftAspNetCoreAuthorizationPackageVersion)" />
+    <Dependency Include="Microsoft.AspNetCore.Components"                              Version="$(MicrosoftAspNetCoreComponentsPackageVersion)" />
+    <Dependency Include="Microsoft.AspNetCore.Components.Browser"                      Version="$(MicrosoftAspNetCoreComponentsBrowserPackageVersion)" />
```

Comment:
Why is this assembly in the shared framework? Isn't this assembly for client scenarios, not servers? Based on the my naive understanding of what this does and the description of this project "Supports rendering...in browsers", it doesn't seem like this belongs in the shared framework.

Reply:
The `.Browser` package is for "rendering into browsers" (i.e., to a DOM, which server-side Razor Components do).

It's not for "executing in browsers" (i.e., Blazor on WebAssembly only).

Reply:
Ok. Should we change the wording of the package description to make this clearer?

https://github.com/aspnet/AspNetCore/blob/b0524058c3e35a2768de34ea56b5ee6afcc968b4/src/Components/src/Microsoft.AspNetCore.Components.Browser/Microsoft.AspNetCore.Components.Browser.csproj#L5

Reply:
How about "Support for rendering ASP.NET Core components for browsers."?